firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -73,6 +78,11 @@
   /* file used for process-level syncronization of generating and persisting fids */
   private static final String LOCKFILE_NAME_GENERATE_FID = "generatefid.lock";
 
+  private static final String IID_SDK_PROP_FILE = "/firebase-iid.properties";
+  private static final String IID_SDK_VERSION_BEFORE_FIS = "20.0.2";
```

Comment:
why only this explicit version?
is this the version of IID since I/O 2019?

Reply:
This is the latest version of IID with no on dependency on FIS SDK.

https://mvnrepository.com/artifact/com.google.firebase/firebase-iid